counter.c

```diff
@@ -411,6 +430,28 @@ int tc_count_packets(struct __sk_buff *skb) {
   return TC_ACT_UNSPEC;
 }
 
+SEC("cgroup/sock_create")
+int cgroup_sock_create(struct bpf_sock *sk) {
+  __u64 cookie = bpf_get_socket_cookie(sk);
+  sockinfo ski = {
+    .pid = bpf_get_current_pid_tgid(),
+    .comm = {0},
+  };
+
+  bpf_get_current_comm(ski.comm, sizeof(ski.comm));
+
+  bpf_map_update_elem(&sock_info, &cookie, &ski, BPF_ANY);
+
+  return ALLOW_SK;
+}
+
+SEC("cgroup/sock_release")
```

Comment:
I don't think we need this debug info, was it left unintentionally in the PR?

Reply:
sorry yes, it should probably be removed for now. Maybe it should remove entries from the map, but I think that would create a data race if traffic is still in-flight.

Reply:
Yep, removal from the map is kind of problematic, although since this is a LRU map with fixed number of elements, I think we are good with keeping old entries until they are overwritten with more recent entries.